test/e2e/utils/fixture/fixture.go

```diff
@@ -0,0 +1,96 @@
+package fixture
+
+import (
+	"context"
+	"fmt"
+	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var F Fixture
+var initialized bool
+
+type Fixture struct {
+	Ctx     context.Context
+	Cli     client.Client
+	K8SCli  *kubernetes.Clientset
+	TopoCli *topologyclientset.Clientset
+	// TestingNS randomly generated for each test and gets deleted when the test ends
+	TestingNS *corev1.Namespace
```

Comment:
let's call it `Namespace`, (or just `NS`), the fact is a testing NS is obvious enough from the context